RealestateApi/Controllers/VisitRequestController.cs

```diff
@@ -76,5 +76,27 @@ public async Task<ActionResult<VisitRequestModel>> PutVisitRequestConfirmationBy
                 ? Ok(response.Result)
                 : Problem(response.ProblemType, string.Join(",", response.AdditionalInformation), (int)HttpCodesEnum.BadRequest);
         }
+
+        /// <summary>
+        /// 
+        /// Https Post Method to create a Visit Request
+        /// 
+        /// </summary>
+        /// <param name="visitRequestData"> Visit Request Data to be created </param>
+        /// 
+        /// Sample Request:
+        /// 
+        ///     POST /api/VisitRequest
+        ///     
+        /// <returns> VisitRequestModel </returns>
+        [HttpPost(Name = "PostVisitRequest")]
+        public async Task<ActionResult<VisitRequestModel>> PostVisitRequestAsync(VisitRequestDto visitRequestData)
+        {
+            var response = await _visitRequestService.PostVisitRequestAsync(visitRequestData);
```

Comment:
Change service method name from PostVisitRequestASync to AddVisitRequestAsync please.